libevm/VMCalls.cpp

```diff
@@ -142,10 +150,18 @@ bool VM::caseCallSetup(CallParameters *callParams)
 		memNeed(m_stack[(1 + m_sp - m_stack) - sizesOffset], m_stack[(1 + m_sp - m_stack) - sizesOffset - 1])
 	);
 	updateMem();
+	updateIOGas();
+
+	// "Static" const already applied. Calculate call gas.
+	auto requestedCallGas = *m_sp;
+	auto maxAllowedCallGas = *m_io_gas - *m_io_gas / 64;
```

Comment:
Again: Please don't use auto here. Note that `m_runGas` is a 64 bit variable and `m_io_gas` is a 256 bit variable. `auto` completely hides this fact and might introduce performance and correctness issues.

Also, perhaps less confusing:
```
auto maxAllowedCallGas = *m_io_gas - (m_schedule->isEIP150() ? *m_io_gas / 64 : 0);
auto callGas = std::min(requestedCallGas, maxAllowedCallGas);
```